src/plugins/validate-semantic/validators/oas3/refs.js

```diff
@@ -29,3 +29,31 @@ export const validateOAS3RefsForRequestBodiesReferenceRequestBodyPositions = ()
       }, [])
     })
 }
+
+export const validateOAS3RefsForHeadersReferenceHeadersPositions = () => sys => {
```

Comment:
This name does not match the validation performed - we're verifying all parameters not just headers. Please change to

```
validateOAS3ParameterRefsReferenceParameterPositions
```
or similar.

Reply:
Done!